src/features/quality-stats/QualityStatsCollector.js

```diff
@@ -231,7 +277,13 @@ class QualityStatsCollector {
 
             this._collectRttData(rtts, statsEntry, report);
             this._collectPacketLossData(pcData, statsEntry, report);
+            this._updateInboundVideoExperience(inboundVideoExperience, statsEntry, report);
         });
+
+        if (inboundVideoExperience.principalVideoSummary) {
```

Comment:
`inboundVideoExperience ` is defined a few lines above the update loop, so here it can't be undefined. The update loop updates its properties (if there are any collected data).